### ISSUE TITLE
controller: cannot access nodex-bmc

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#369]
**version**:  r5_fixes f3f13d1
**hardware**: amaster

From controller, i cannot rpower nodexxx reset

timeout error.

```
[root@controller ~]# ping node001-bmc
PING node001-bmc (10.148.0.1) 56(84) bytes of data.

```

**Comment by hansthen** [comment]
Fixed in 16b04269cf03ae15189bbe51b5c7ab8b7952035c. As a workaround I added the BMC ip address directly to the interface.

**Comment by themisa** [comment]
Fixed in r5/f5_fixes 

**Version:** r5_fixes e5f4623

```
[root@controller() ~]# ping node001-bmc
PING node001-bmc (10.148.0.1) 56(84) bytes of data.
64 bytes from node001-bmc (10.148.0.1): icmp_seq=1 ttl=64 time=1.38 ms
64 bytes from node001-bmc (10.148.0.1): icmp_seq=2 ttl=64 time=1.37 ms
```
